opm/output/data/Wells.hpp

```diff
@@ -538,6 +541,12 @@ namespace Opm {
         buffer.write(this->thp);
         buffer.write(this->temperature);
         buffer.write(this->control);
+
+        {
+            const auto status = ::Opm::Well::Status2String(this->dynamicStatus);
+            buffer.write(status);
```

Comment:
Are we using strings?

Reply:
We are.  It's the only serialisation method available for `enum class` objects unless we essentially want to duplicate the logic of `Status2String` and `StatusFromString` in terms of `int`s.

Reply:
OK - I would have thought you could get away with `static_cast<int>()`?